tests/foreman/cli/test_activationkey.py

```diff
@@ -1276,6 +1276,55 @@ def test_bugzilla_1111723(self):
 
         self.assertEqual(new_activation_key['name'], name)
 
+    def test_remove_host_collection_by_id(self):
+        """@Test: Test that hosts associated to Activation Keys can be removed
+        using id of that host collection
+
+        @Feature: Activation key - Host
+
+        @Steps:
+
+        1. Create Activation key
+        2. Create host collection
+        3. Associate host collection to Activation key
+        4. Remove host collection associated to Activation key using id of that
+        collection
+
+        @Assert: Host collection successfully removed from activation key
+
+        """
+        try:
+            activation_key = self._make_activation_key({
+                u'organization-id': self.org['id'],
+            })
+            new_host_col = make_host_collection({
+                u'name': gen_string('alpha'),
+                u'organization-id': self.org['id'],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        result = ActivationKey.add_host_collection({
+            u'name': activation_key['name'],
+            u'host-collection-id': new_host_col['id'],
+            u'organization': self.org['name'],
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0, result.stderr)
```

Comment:
We already have `result.stderr` in the logs, not need to add it as the message. Please drop in other places too.